test/LaunchDarkly.Tests/OperatorTest.cs

```diff
@@ -176,6 +176,11 @@ public void Apply_Contains_SupportedTypes_ReturnsFalse(object userValue, object
         [Theory]
         [InlineData(10, 11)]

```

Comment:
All of these additional tests do the same thing.  Edge cases with combinations of different value types